Src/library/cURL/interface/curl_easy_externals_i.e

```diff
@@ -355,7 +365,7 @@ feature {NONE} -- C externals
 
 	c_getinfo (a_api: POINTER; a_curl_handle: POINTER; a_opt: INTEGER; a_data: POINTER): INTEGER
 			-- C implementation of `curl_easy_getinfo'.
-			-- Declared as curl_easy_setopt ().
+			-- Declared as curl_easy_setopt .
```

Comment:
is it really "Declared as curl_easy_setopt"?? setopt ?
in fact I don't understand this comment here, and elsewhere.